src/WebJobs.Extensions.Http/ClaimsPrincipalBindingProvider.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    /// <summary>
+    /// This provider provides a binding to Type <see cref="ClaimsPrincipal"/>.
+    /// </summary>
+    internal class ClaimsPrincipalBindingProvider : IBindingProvider
+    {
+        private static Task<IBinding> nullBinding = Task.FromResult<IBinding>(null);
```

Comment:
>static [](start = 16, length = 6)

style nit: Make this readonly and capitalize it

Reply:
Interestingly, until I added readonly, the project wouldn't build because of a style error if I used uppercase with a private error. Good to know adding readonly avoids that. 

---
In reply to: [174630304](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/373#discussion_r174630304) [](ancestors = 174630304)